packages/inspection-capture-web/src/PhotoCapture/PhotoCaptureHUD/PhotoCaptureHUDElementsAddPartSelectShot/PhotoCaptureHUDElementsAddPartSelectShot.tsx

```diff
@@ -0,0 +1,64 @@
+import { getLanguage, useMonkAppState, useMonkTheme, vehiclePartLabels } from '@monkvision/common';
+import { Button, VehiclePartSelection } from '@monkvision/common-ui-web';
+import { VehiclePart } from '@monkvision/types';
+import { useEffect, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { PhotoCaptureHUDElementsProps } from '../PhotoCaptureHUDElements/PhotoCaptureHUDElements.model';
+import { usePhotoCaptureHUDElementsAddPartSelectShotStyle } from './usePhotoCaptureHUDElementsAddPartSelectShotStyle';
+
+type PhotoCaptureHUDElementsAddPartSelectShotProps = Pick<
+  PhotoCaptureHUDElementsProps,
+  'onAddDamageParts'
+> & {
+  onCancel: PhotoCaptureHUDElementsProps['onCancelAddDamage'];
+  partSelectState: PhotoCaptureHUDElementsProps['addDamagePartSelectState'];
+};
+export function PhotoCaptureHUDElementsAddPartSelectShot({
+  onCancel,
+  onAddDamageParts,
+  partSelectState,
+}: PhotoCaptureHUDElementsAddPartSelectShotProps) {
+  const { vehicleType } = useMonkAppState();
+  const { palette } = useMonkTheme();
+  const [selectedParts, setSelectedParts] = useState<VehiclePart[]>([]);
+  const { i18n, t } = useTranslation();
+  const style = usePhotoCaptureHUDElementsAddPartSelectShotStyle();
+  useEffect(() => onAddDamageParts(selectedParts), [selectedParts]);
+  if (partSelectState === 'image-capture') {
+    return null;
+  }
```

Comment:
blank lines above and under this if statement

Reply:
this is removed